odata-csdl/Appendix.md

```diff
@@ -152,6 +152,254 @@ https://www.ogc.org/standard/sfa/.
 $$$isec$$$
 :::
 
+-------
+
+# Appendix ##asec Model elements, properties and relationships
+
+## ##subasec Overview
+
+This appendix describes the model elements, their properties and relationships of the abstract OData model. The XSD schema is the normative document for the model elements and their properties. Yet, because of the limitations of XSD schema, it only contains the parent-child relationships of the model elements. For example the type of a property is represented as a string in XML and JSON, a string of the qualified name of the type. In the abstract model, it is a reference to the element that is the type.
```

Comment:
```suggestion
This appendix describes the model elements and their properties and relationships of the abstract OData model. The XSD schema is the normative document for the model elements and their properties. Yet, because of the limitations of XSD schema, it only contains the parent-child relationships of the model elements. For example the type of a property is represented as a string in XML and JSON, a string of the qualified name of the type. In the abstract model, it is a reference to the element that is the type.
```
Also better have one sentence per source line to make reviews easier.

Reply:
I don't know what you mean by source line. line in the Appendix.md ?